CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-02-08
+
+This release is rolling out to all regions - ETA for conclusion 2021-02-17 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on May 31st, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Starting on the week of 15 February 2021 (Azure China Cloud and Azure Government Cloud users will get this update in the following weeks), we will upgrade AKS clusters Calico network policy from Calico version v3.8.9 to v3.17.1. This upgrade will cause a breaking change to the default behavior of all-interfaces Host Endpoints. For customers that use Host Endpoints, and only these, this version brings a change. Please follow our [guidance](https://github.com/Azure/AKS/issues/2089) to apply the appropriate label and Global Network Policy if you want to keep the v3.8.9 default behavior of all-interfaces Host Endpoints.
+
+### Release Notes
+
+* Preview Features
+  * AKS now supports Private Clusters created with a custom DNS zone (BYO DNS zone). Read more [here](https://docs.microsoft.com/azure/aks/private-clusters#configure-private-dns-zone)
+* Behavioral Change
+  * The process of creating a Persistant Volume will be delayed until a pod is created. Allowing the Persistant Volume to be created in the same zone as the pod.
```

Comment:
*persistent

Reply:
And be specific here you can link to our docs here:
https://docs.microsoft.com/en-us/azure/aks/azure-disk-csi#create-a-custom-storage-class

Which should be updated to the opposite now that the default behavior has been reversed.
@mlearned @zr-msft @Nickomang

Reply:
Resolved